src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/overlays/ExternalLinkTypeOverlay.js

```diff
@@ -170,8 +170,17 @@ class ExternalLinkTypeOverlay extends React.Component<LinkTypeOverlayProps> {
             target,
             title,
             href,
+            options,
         } = this.props;
 
+        if (!options || !options.targets) {
+            throw new Error('The ExternalLinkTypeOverlay needs atleast targets as options in order to work!');
```

Comment:
This is a BC break. If there are no targets given we should render all targets currently.

In 3.0 we can decide if no targets are given we should maybe just not render the target field here.